### PR TITLE
code health Documentation updates

### PR DESCRIPTION
**Pull request by pauli-oracle**
- [x] documentation is added or updated

Fix capitilistion of list items.
Wrap long lines.
Add full stops to the ends of sentances.

### CODE REVIEW

File: doc/man1/ocsp.pod [hunk 1]
```diff
@@ -299,16 +300,17 @@ digest used by subsequent certificate identifiers.
 
 =item B<-index indexfile>
 
-B<indexfile> is a text index file in B<ca> format containing certificate revocation
-information.
+B<Indexfile> is a text index file in B<ca> format containing certificate
```
Comment (mattcaswell) [inline doc/man1/ocsp.pod]: Not sure about this one. This is a parameter name, so surely it should be a lower case to match how it is written above?

File: doc/man1/ciphers.pod [hunk 2]
```diff
@@ -346,7 +346,7 @@ Cipher suites using SHA1.
 
 =item B<SHA256>, B<SHA384>
 
-Ciphersuites using SHA256 or SHA384.
+Cipher suites using SHA256 or SHA384.
```
Comment (mattcaswell) [inline doc/man1/ciphers.pod]: We seem to be inconsistent about this. I get 438 matches for "ciphersuite" in the codebase and 128 for "cipher suite". Personally I prefer the former.

File: doc/man1/pkeyutl.pod [hunk 3]
```diff
@@ -144,12 +143,12 @@ hex dump the output data.
 
 =item B<-asn1parse>
 
-asn1parse the output data, this is useful when combined with the
+Asn1parse the output data, this is useful when combined with the
```
Comment (mattcaswell) [inline doc/man1/pkeyutl.pod]: This kind of doesn't make sense. "asn1parse" is an option name and shouldn't have a capital letter. Here though it seems to be being used as a verb.

File: doc/man1/ocsp.pod (hunk 1 above)
Comment (pauli-oracle) [inline doc/man1/ocsp.pod]: I've reworded the sentence so it doesn't start with indexfile.

File: doc/man1/rsautl.pod [hunk 4]
```diff
@@ -44,56 +44,56 @@ if this option is not specified.
...
 
 =item B<-hexdump>
 
-hex dump the output data.
+Hex dump the output data.
 
 =item B<-asn1parse>
 
-asn1parse the output data, this is useful when combined with the
+Asn1parse the output data, this is useful when combined with the
```
Comment (mattcaswell) [inline doc/man1/rsautl.pod]: Again here

File: doc/man1/s_client.pod [hunk 5]
```diff
@@ -473,59 +473,57 @@ will be used.
...
-The B<protocols> list is a
-comma-separated protocol names that the client should advertise
-support for. The list should contain most wanted protocols first.
-Protocol names are printable ASCII strings, for example "http/1.1" or
-"spdy/3".
+These flags enable the Enable the Application-Layer Protocol Negotiation
+or Next Protocol Negotiation extension, respectively. ALPN is the IETF
+standard and replaces NPN.
+The B<protocols> list is a comma-separated protocol names that the
+client should advertise support for. The list should contain most wanted
```
Comment (pauli-oracle) [inline doc/man1/s_client.pod]: This is a re-wrapping of the text without changes.  I did think about dropping the "Enable the".

The second sentence is bad.  This text is repeated in s_server.pod.

File: doc/man1/ciphers.pod (hunk 2 above)
Comment (pauli-oracle) [inline doc/man1/ciphers.pod]: I've changed to _ciphersuite_, although that doesn't appear in the dictionaries I've checked (OED and Webster's).  This isn't an issue in code, but shouldn't the documentation use dictionary words where possible?

Comment (mattcaswell) [inline doc/man1/ciphers.pod]: Well, as in many things there is a whole vocabulary which is fairly specialist and not likely to appear in a dictionary. In this case the word comes from the RFC - although both cipher suite and ciphersuite seems to be used.

Comment (mattcaswell) [inline doc/man1/ciphers.pod]: Anyway...I don't have a strong opinion, but I'd like to be consistent.

File: doc/man1/pkeyutl.pod (hunk 3 above)
Comment (pauli-oracle) [inline doc/man1/pkeyutl.pod]: Any suggestion as to a rewording here?  I'm getting a mental blank.

Comment (mattcaswell) [inline doc/man1/pkeyutl.pod]: "Parse the ASN.1 output data..."?

Comment (pauli-oracle) [inline doc/man1/pkeyutl.pod]: Thanks, that's nice and simple.

File: doc/man1/ciphers.pod (hunk 2 above)
Comment (pauli-oracle) [inline doc/man1/ciphers.pod]: All the documentation should be consistent now.

I've not touched the NEWS, INSTALL or CHANGES files at the top level.  The rest of the references to _ciphersuite_ are in code and tests.

**Comment by levitte** [comment]
Merged. c4de074e6385a86a43a30fee574e77f9dcabb022

**Pull request closed without merging**